### PR TITLE
resolve #15 - Check how using pull request to push in branch with verification - Update README.md

### LINKED ISSUE #15: Check how using pull request to push in branch with verification

### PR DESCRIPTION
**Pull request by adelramezani**
The change after creating ruleset and enabling Require a pull request before merging! (SUCCEED!)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,3 +4,5 @@ Testing docker-compose with having 1 oracle database and 1 wavemaker application
 The change made by other contributor than owner! (SHOULD NOT)
 
 The change after dis-allow merge commits! (SHOULD NOT)
+
+The change after creating ruleset and enabling Require a pull request before merging! (SHOULD NOT)
```
Comment (adramazany) [inline README.md]: Change the SHOULD NOT to SUCCEED

**Review by adramazany — CHANGES_REQUESTED** [review]
Please do the comments

File: README.md (hunk 1 above)
Comment (adelramezani) [reply README.md]: Done
I mean, changed to suceed

**Review by adramazany — APPROVED** [review]
Approve

**Comment by adramazany** [comment]
close #15

**Pull request merged**